src/ocm/components/clusterConfiguration/networkConfiguration/NetworkConfiguration.tsx

```diff
@@ -154,6 +167,8 @@ const NetworkConfiguration = ({
         setFieldValue('machineNetworks', [], false);
       }
     } else {
+      //We need to set umn=false in the server when "Cluster managed networking" is selected
+      void resetUserManagedNetworking(cluster.id, cluster.tags);
```

Comment:
Wouldn't this patch happen even in the case that the cluster already has the `userManagedNetworking` set to false?

I think it's preferable to do it in the `onChange` of the RadioButton, when the newly selected value is "cluster managed network". Then it would only be done at that particular moment.